packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -220,6 +220,10 @@ class UserMessages {
       "matching '$deviceId'";
   String get flutterNoDevicesFound => 'No devices found';
   String get flutterNoSupportedDevices => 'No supported devices connected.';
+  String flutterMissPlatformProject(String flutterProject) =>
+      'Is your project missing a platform project in your project $flutterProject'
+      '\nConsider running "flutter create ." to create one.';
+  String get flutterFoundButUnSupportedDevices=> 'Follow devices are founded but not support in this project: ';
```

Comment:
Maybe `The following devices were found, but they aren't supported in this project:`